src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-ffpyplayer.py

```diff
@@ -0,0 +1,19 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2021 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+from PyInstaller.utils.hooks import eval_statement, collect_submodules
+
+hiddenimports = collect_submodules("ffpyplayer")
+binaries = []
+# ffpyplayer has an internal variable tells us where the libraries it was using
+for bin in eval_statement("import ffpyplayer; print(ffpyplayer.dep_bins)"):
```

Comment:
I think long-term it would make sense to use `get_module_attribute` here. At the moment, though, 4.x returns string-ified value, while 5.x returns native-type version (presumably a list), so explicitly handling both might not be worth the hassle.